ptp/components/utils/word_mappings.py

```diff
@@ -135,3 +135,21 @@ def save_word_mappings_to_csv_file(logger, folder, filename, word_to_ix, fieldna
             writer.writerow({fieldnames[0]:k, fieldnames[1]: v})
 
     logger.info("Saved mappings of size {} to file '{}'".format(len(word_to_ix), file_path))
+
+def pad_list(l: list, length: int, value = 0):
```

Comment:
1) maybe the name should be pad_truncate_list, that will reflect more its behavior?

2) not sure whether word_mappings.py is the right place... but please leave it here for now.

Reply:
Yeah since the function is used both in `SentenceEmbeddings` and `SentenceIndexer`, I though that it should be somewhere in a "utils" collection of static functions.